setup.cfg

```diff
@@ -47,8 +47,13 @@ install_requires =
 [options.extras_require]
 benchmarks =
     asv[virtualenv] ~= 0.6.0, < 0.6.2
+test =
```

Comment:
I would prefer it if `pytest` was explicitly listed here.

Reply:
sure, I just missed it, thanks!

```suggestion
test =
    pytest
```